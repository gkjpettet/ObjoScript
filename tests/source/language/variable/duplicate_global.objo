var g1 = 1
var g1 = 2 # expect error: "Redefined global variable `g1`."