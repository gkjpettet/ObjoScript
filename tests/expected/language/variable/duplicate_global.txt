Redefined global identifier `g1`.